service/controller/key/key_test.go

```diff
@@ -1190,17 +1190,10 @@ func Test_VersionBundleVersion(t *testing.T) {
 
 func Test_BucketObjectName(t *testing.T) {
 	t.Parallel()
-	customObject := v1alpha1.AWSConfig{
-		Spec: v1alpha1.AWSConfigSpec{
-			VersionBundle: v1alpha1.AWSConfigSpecVersionBundle{
-				Version: "0.1.0",
-			},
-		},
-	}
 	role := "worker"
 
-	e := fmt.Sprintf("version/0.1.0/cloudconfig/%s/worker", CloudConfigVersion)
-	a := BucketObjectName(customObject, role)
+	e := fmt.Sprintf("ignition/worker")
```

Comment:
Changing this path might have unforseen consequences - we should definitely test cluster upgrades with this.

Reply:
Any ideas about what might happen? Once a cluster is upgrading, there's no reason to keep old ignition files around, right? I'm thinking if a cluster is autoscaling up while upgrading, it might have some nodes start in the old launch configuration with the new ignition, but the node would be rolled soon anyways.